### PR TITLE
Kelly 6 save off artifact

### PR DESCRIPTION
**Pull request by KellyRyanGlobal**
1. Pull the branch `kelly-6-save-off-artifact`
2. Run the instructions in sample test plan under 3.3.2 procedure
3. replace step 11 with `kelly-6-save-off-artifact`
4. Save and build the application
5. Verify the test turns green as the procedure states
6. 
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,12 +151,15 @@ To build logger locally in your workstation, run the following command
     1. `g++ test.cc -o test.exe`
     
 
-## Build logger in docker
+## Build logger in docker local docker container
```
Comment (AnnMillikin) [inline README.md:154]: What is this line saying?

File: jenkins/Jenkinsfile-1 [hunk 2]
```diff
@@ -0,0 +1,12 @@
+pipeline {
+  agent {
```
Comment (AnnMillikin) [inline jenkins/Jenkinsfile-1:2]: I didn't test this file as part of the steps.  Was I supposed to?

**Review by AnnMillikin — COMMENTED** [review]
Please answer the questions, thanks!

File: README.md (hunk 1 above)
Comment (KellyRyanGlobal) [reply README.md:154]: That it is building the logger tester app in a local docker container

File: jenkins/Jenkinsfile-1 (hunk 2 above)
Comment (KellyRyanGlobal) [reply jenkins/Jenkinsfile-1:2]: no this is a sample jenkinsfile I can remove it

**Review by AnnMillikin — APPROVED** [review]
Thanks!

**Pull request merged**